frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/utilization-card/utils.tsx

```diff
@@ -13,13 +18,8 @@ export const humanizeIOPS: Humanize = (value) => {
 };
 
 export const humanizeLatency: Humanize = (value) => {
-  const humanizedNumber = humanizeNumber(value);
-  const unit = 'ms';
-  return {
-    ...humanizedNumber,
-    string: `${humanizedNumber.value} ${unit}`,
-    unit,
-  };
+  const humanizedNumber = humanizeSeconds(secondsToNanoSeconds(value), null, 'ms');
```

Comment:
Is latency in the domain of nanoseconds or milliseconds? 
IMO it'd be better to rename the variable to humanizeTime.